src/pages/comparison/comparison.tsx

```diff
@@ -369,9 +387,17 @@ export const Comparison = ({ mode }: ComparisonPageProps) => {
    * @returns {string[]} Sublayer urls or tileset url.
    * TODO Add filtration mode for sublayers which were selected by user.
    */
-  const getFlattenedSublayers = async (tilesetUrl) => {
+  const getFlattenedSublayers = async (tilesetUrl, side: "left" | "right") => {
```

Comment:
I guess it will be good to have `enum ComparisonSide {left, rigth}`

Reply:
Yeap. Lets do it in a separate PR. There are already come functions written previously with `side` argument.

Reply:
Ok